src/lib.rs

```diff
@@ -118,6 +118,14 @@ macro_rules! implement_common {
                 $name(value)
             }
 
+            /// Creates a new variable without checking if the variable is representable by this type.
```

Comment:
```suggestion
            /// Creates a new instance without checking if the value is representable by this type.
```

Reply:
I based it on the documentation of the `new` function, but can also change it tomorrow.